include/mysql/plugin.h

```diff
@@ -995,6 +995,10 @@ void thd_kill(unsigned long id);
 */
 int thd_get_ft_query_extra_word_chars(void);
 
+using ssl_reload_callback_t = void(*)();
```

Comment:
Just in case, you may want to make the signature `void(*)(void*)` for future possible extensions without changing the interface, so that the callback could be invoked with some `void *cb_data`;

Reply:
This sounds like similar to why I added an int return type to the registration function, even if currently it always returns 0 - I'm not sure if I remove one futureproofing, adding a different one makes sense.